### PR TITLE
docs(MVUX): Add how-to use state

### PR DESCRIPTION
**Pull request by weitzhandler**
## PR Type

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: doc/Learn/Tutorials/Authentication/HowTo-Authentication.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ uid: Learn.Tutorials.Authentication.HowToAuthentication
 `Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to use one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```
Comment (nickrandolph) [inline doc/Learn/Tutorials/Authentication/HowTo-Authentication.md:6]: Please revert changes to this file in this PR - if these changes are required, please push to a different PR

File: doc/Overview/Reactive/How-tos/MvuxProject.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-uid: Overview.Reactive.HowTos.MvuxProject
+uid: Overview.Reactive.MvuxProject
 ---
 
 # How to set up an MVUX project
 
 In this tutorial you'll learn how to set up an Uno Platform project to use MVUX. 
 
-1. Make sure your environment is set up properly by using [uno check](xref:UnoCheck.UsingUnoCheck).
+1. Make sure your environment is set up properly by using [uno check](/docs/articles/external/uno.check/doc/using-uno-check.html).
```
Comment (nickrandolph) [inline doc/Overview/Reactive/How-tos/MvuxProject.md:9]: Revert this change - there's a PR open on unocheck to add the id to make the xref work

File: doc/Overview/Reactive/How-tos/SimpleFeed.md [hunk 3]
```diff
@@ -1,30 +1,31 @@
 ---
-uid: Overview.Reactive.HowTos.SimpleFeed
+uid: Overview.Reactive.SimpleFeed
```
Comment (nickrandolph) [inline doc/Overview/Reactive/How-tos/SimpleFeed.md:2]: If we're adjusting the uid, can we replace Reactive with MVUX

File: doc/Overview/Reactive/How-tos/SimpleFeed.md [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
 ---
 
 # How to create a simple feed
 
-In this tutorial you will learn how to create an MVUX project and use a combination of a feed (`IFeed<T>`) and the `FeedView` control to load and display data coming from a service.
+In this tutorial you will learn how to create an MVUX project and use a combination of a feed (`IFeed<T>`)
+and the `FeedView` control to asynchronously load and display data coming from a service.
```
Comment (nickrandolph) [inline doc/Overview/Reactive/How-tos/SimpleFeed.md:8]: I get that md supports random line breaks but I don't like this and would prefer to just use an editor that supports word wrapping!

File: doc/Overview/Reactive/How-tos/SimpleState.md [hunk 5]
```diff
@@ -26,7 +25,13 @@ In this tutorial you will learn how to create an MVUX project and basic usage of
...
 
-    public class HallCrowdednessService
+    public interface IHallCrowdednessService
+    {
+        ValueTask<HallCrowdedness> GetHallCrowdednessAsync(CancellationToken ct);
+        ValueTask SetHallCrowdednessAsync(HallCrowdedness crowdedness, CancellationToken ct);
+    }
+
+    public class HallCrowdednessService : IHallCrowdednessService
     {
```
Comment (nickrandolph) [inline doc/Overview/Reactive/How-tos/SimpleState.md:35]: Add delay to both Get and Set methods to simulate service calls

File: doc/Overview/Reactive/How-tos/SimpleFeed.md (hunk 3 above)
Comment (weitzhandler) [reply doc/Overview/Reactive/How-tos/SimpleFeed.md:2]: Furthermore, it was clashing with the already existing Reactive docs which we've moved to Reference.

File: doc/toc.yml [hunk 6]
```diff
@@ -67,8 +67,10 @@
           href: Overview/Reactive/Overview.md
         - name: How to create an MVUX project
           href: Overview/Reactive/How-tos/MvuxProject.md
-        - name: How to create a simple feed
+        - name: How to create a simple Feed
```
Comment (nickrandolph) [inline doc/toc.yml:70]: Remove "simple" from these titles

File: doc/Overview/Reactive/How-tos/SimpleFeed.md (hunk 4 above)
Comment (weitzhandler) [reply doc/Overview/Reactive/How-tos/SimpleFeed.md:8]: The reason I like wrapping MDs in the file, is for when it's viewed on GH it's irritating.
I can fix them all if you want me to. I'll def avoid from now on.

File: doc/Overview/Mvux/How-tos/SimpleState.md [hunk 7]
```diff
@@ -1,23 +1,22 @@
...
-# How to create a simple state
+# How to create a state with a command
 
-In this tutorial you will learn how to create an MVUX project and basic usage of a state (`IState<T>`) and the `FeedView` control.
+In this tutorial you will learn how to create an MVUX project and basic usage of a state (`IState<T>`)
+and the `FeedView` control, to asynchronously load, display and manipulate data from and to an service.
 
- - For our data we're going to create a service that retrieves and updates a single value
+ - The data is provided by a service that asynchronously retrieves and updates a single value.
  that determines the crowdedness of a wedding hall, via a 'remote' service.
```
Comment (nickrandolph) [inline doc/Overview/Mvux/How-tos/SimpleState.md:11]: Is there an extra fullstop here, or should this be two sentences?

File: doc/Overview/Mvux/How-tos/SimpleState.md [hunk 8]
```diff
@@ -61,59 +65,59 @@ In this tutorial you will learn how to create an MVUX project and basic usage of
...
+and enable manipulating the data, as opposed to Feeds, which doesn't.
 
 > [!TIP]
 > An `IFeed<T>` as well as `IState<T>` are awaitable,
 > meaning that to get the value of the feed you would do the following in the model:  
-> 
+>
 > ```c#
 > HallCrowdedness hallCrowdedness = await this.HallCrowdedness;
 > ```
```
Comment (nickrandolph) [inline doc/Overview/Mvux/How-tos/SimpleState.md:113]: Is there a section missing here that walks through data binding the state?

**Review by nickrandolph — APPROVED** [review]
